application/common/models/Mfa.php

```diff
@@ -342,6 +412,8 @@ public static function create(int $userId, string $type, string $label = null, s
                 ]);
                 throw new ServerErrorHttpException("Unable to update MFA record", 1507904194);
             }
+
+            $mfa->createWebauthn($results, $user);
```

Comment:
Why was this call added? Just from reviewing the code, I don't see an obvious reason, but I'm half-skimming this part, so I probably just missed it.